### DISCUSSION TITLE
Safari not showing in the list of apps

### DISCUSSION THREAD

**Discussion by GrayJack** [rxhanson/Charmstone-Community#22] [Q&A]
I just installed the update 0.26 and try to change some app locations and noticed that safari is not in the list

**Comment by rxhanson** [comment]
Ah, you're right, thanks. Something must have changed macOS side here that I didn't catch. Until a fix makes it through App Store review, you can just drag and drop Safari in from Finder, either into the prefs window or into Charmstone itself.

**Reply by rxhanson** [reply]
Indeed, there was a change in Ventura, and Safari is now in a different location but symlinked into the Applications directory. The new location is /System/Cryptexes/App/System/Applications/
Here's some info about the directory: https://eclecticlight.co/2022/11/16/cryptex-how-a-custom-iphone-is-changing-macos-updates/

The bug fix will be in v0.27 hopefully out shortly.
